pkg/operator/controller/controller_dns_configmap.go

```diff
@@ -196,27 +206,31 @@ func desiredDNSConfigMap(dns *operatorv1.DNS, clusterDomain string, caBundleRevi
 	pTTL, nTTL := coreDNSCache(dns)
 
 	corefileParameters := struct {
-		ClusterDomain       string
-		Servers             interface{}
-		UpstreamResolvers   operatorv1.UpstreamResolvers
-		PolicyStr           func(policy operatorv1.ForwardingPolicy) string
-		LogLevel            string
-		CABundleRevisionMap map[string]string
-		CABundleFileName    string
-		LameDuckDuration    time.Duration
-		PositiveTTL         uint32
-		NegativeTTL         uint32
+		ClusterDomain             string
+		Servers                   interface{}
+		UpstreamResolvers         operatorv1.UpstreamResolvers
+		PolicyStr                 func(policy operatorv1.ForwardingPolicy) string
+		LogLevel                  string
+		CABundleRevisionMap       map[string]string
+		CABundleFileName          string
+		LameDuckDuration          time.Duration
+		PositiveTTL               uint32
+		NegativeTTL               uint32
+		OCPDNSNameResolver        bool
+		DNSNameResolverNamepsaces []string
 	}{
-		ClusterDomain:       clusterDomain,
-		Servers:             dns.Spec.Servers,
-		UpstreamResolvers:   upstreamResolvers,
-		PolicyStr:           coreDNSPolicy,
-		LogLevel:            coreDNSLogLevel(dns),
-		CABundleRevisionMap: caBundleRevisionMap,
-		CABundleFileName:    caBundleFileName,
-		LameDuckDuration:    lameDuckDuration,
-		PositiveTTL:         pTTL,
-		NegativeTTL:         nTTL,
+		ClusterDomain:             clusterDomain,
+		Servers:                   dns.Spec.Servers,
+		UpstreamResolvers:         upstreamResolvers,
+		PolicyStr:                 coreDNSPolicy,
+		LogLevel:                  coreDNSLogLevel(dns),
+		CABundleRevisionMap:       caBundleRevisionMap,
+		CABundleFileName:          caBundleFileName,
+		LameDuckDuration:          lameDuckDuration,
+		PositiveTTL:               pTTL,
+		NegativeTTL:               nTTL,
+		OCPDNSNameResolver:        dnsNameResolverEnabled,
+		DNSNameResolverNamepsaces: []string{DefaultDNSNameResolverNamespace},
```

Comment:
Don't we want the namespaces as a part of the operator config along with the bool about enabled or not?

Reply:
I'll add the namespace as part of the config.